### DISCUSSION TITLE
Build netcard ipxe rom with predefined settings. how?

### DISCUSSION THREAD

**Discussion by dipol0** [ipxe/ipxe#254] [Q&A]
I need build ipxe for netcard rom. All build fine but i need HARD defined ip,netmask,next-server,sanbot... etc parameters. How and where in SRC i need setup tis parameters for build ipxe netcard rom with this HARD predefined settings ??? Sorry for my ENG. Tnx.

**Comment by mcb30** [comment]
You can use an embedded script (https://ipxe.org/embed).  For example, create `myscript.ipxe` containing:

```
#!ipxe

ifopen net0
set net0/ip 192.168.0.10
set net0/netmask 255.255.255.0
set net0/gateway 192.168.0.1
set dns 8.8.8.8
chain http://...........
```

and then add `EMBED=myscript.ipxe` when you build iPXE.
